lib/suse/connect/yast.rb

```diff
@@ -100,6 +100,28 @@ def system_migrations(products, client_params = {})
           Client.new(config).system_migrations(products)
         end
 
+        # Returns installed and activated products on the system
+        # @param [Hash] client_params parameters to instantiate {Client}
+        # @return [Array <Connect::Product>] the list of system products
+        def system_products(client_params = {})
+          config = SUSE::Connect::Config.new.merge!(client_params)
+          status = Status.new(config)
```

Comment:
you can squash it it one liner: `Status.new(config).system_products` like we do it in other methods for example: `Client.new(config).system_migrations(products)`